packages/start-slicemachine/src/StartSliceMachineProcess.ts

```diff
@@ -124,6 +124,9 @@ export class StartSliceMachineProcess {
 				userID: profile.shortId,
 				intercomHash: profile.intercomHash,
 			});
+			try {
+				await this._setDevEnvAsActiveEnvironment();
+			} catch (_) {}
```

Comment:
What do you think about displaying a console.error only on DEV mode for us, so we have a warning in DEV mode this fail one day. I guess it should not throw an error if you just don't have a dev env, it will throw if there is a real problem, no?